bin/compile

```diff
@@ -122,8 +122,8 @@ install_bins() {
     mcount "version.iojs.$node_version"
   else
     warn_node_engine "$node_engine"
-    install_nodejs "$node_engine" "$BUILD_DIR/.heroku/node"
-    install_npm "$npm_engine" "$BUILD_DIR/.heroku/node" $NPM_LOCK
+    monitor "install-node-binary" install_nodejs "$node_engine" "$BUILD_DIR/.heroku/node"
+    monitor "install-npm-binary" install_npm "$npm_engine" "$BUILD_DIR/.heroku/node" $NPM_LOCK
```

Comment:
`monitor` used to break on this, but that was fixed by https://github.com/heroku/heroku-buildpack-nodejs/pull/592